app/modules/asset/service.server.ts

```diff
@@ -1131,6 +1147,15 @@ export const getPaginatedAndFilterableAssets = async ({
       where: { organizationId, id: { in: tagsIds } },
     }),
     db.tag.count({ where: { organizationId } }),
+    // locations
+    db.location.findMany({
+      where: { organizationId, id: { notIn: locationIds } },
+      take: getAllEntries.includes("location") ? undefined : 12,
+    }),
+    db.location.findMany({
```

Comment:
Not sure why we need the 2 queries. Am i missing something?

Reply:
So here,
First query is getting the selected item and second query is getting all the items except the selected.
This is to make sure that when a user has selected a value in dropdown, it is always present in initially loaded dropdown items.